.github/workflows/test_and_release.yml

```diff
@@ -10,28 +10,37 @@ jobs:
       - uses: actions/checkout@v2
         with:
           persist-credentials: false
+
       - uses: actions/cache@v1
         with:
           path: ~/.npm
           key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
           restore-keys: |
             ${{ runner.os }}-node-
+
       - uses: actions/setup-node@v1
         with:
           node-version: 12
           registry-url: https://npm.pkg.github.com/
           scope: '@goproperly'
+
       - name: Install
         run: npm ci
         env:
           NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build
+        run: npm run build
+
       - name: Lint
         run: npm run lint
+
       - name: Test
         run: npm run test
-#      - name: Release & Publish
-#        if: github.ref == 'refs/heads/main'
-#        env:
-#          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
-#          NODE_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
-#        run: npm run release
+
+     - name: Release & Publish
```

Comment:
Looks like the indentation a bit off here?

(Maybe we should configure `lint-staged` to run `prettier` on `yml` files? I think it supports it.)

Reply:
:thumbsup: Yep prettier should be able to do that. Will update.